nox/sessions.py

```diff
@@ -131,9 +131,9 @@ def python(self) -> Optional[Union[str, Sequence[str], bool]]:
         return self._runner.func.python
 
     @property
-    def bin(self) -> Optional[str]:
-        """The bin directory for the virtualenv."""
-        return self.virtualenv.bin
+    def bin_paths(self) -> Optional[List[str]]:
```

Comment:
Nox has avoided breaking API changes for a long time. Can you add a `bin` alias that returns the first entry in the list to maintain backwards compatibility?

Reply:
Sure. Shall I do it for ProcessEnv , CondaEnv and VirtualEnv too ?

Reply:
Yes.

Reply:
Done.